silkshopy-ecommerce/src/pages/Authentication/Login.jsx

```diff
@@ -1,22 +1,68 @@
 import "./Authentication.css";
 import { Link } from "react-router-dom";
+import { useAuth } from "../../contexts/context/authentication-context";
+import { useEffect, useState } from "react";
+import axios from "axios";
+import { useNavigate } from "react-router";
 
 
 const Login = () => {
+
+    const navigate = useNavigate();
+    const [userData, setUserData] = useState({email : "", password : ""});
+    const { authState, authDispatch } = useAuth();
+
+    // Dummy data for guest credential
+    const dummyData = {
+        email : "adarshbalak@gmail.com",
+        password : "adarshBalaki123"
+    }
+
+    // Use guest credential click handler
+    const addDummyUser = (e) => {
+        e.preventDefault();
+        setUserData(dummyData);
+    }
+
+    // User input values click handler
+    const addUserEmail = (e) => {
+        setUserData((pre) => ({ ...pre, email : e.target.value }))
+    }
+
+    const addUserPassword = (e) => {
+        setUserData((pre) => ({ ...pre, password : e.target.value }))
+    }
+
```

Comment:
You can use one function to update both ```email``` and ```password```.
Use ```[e.target.name] : e.target.value```

Reply:
yes, i will do that